src/unix-manager.c

```diff
@@ -695,7 +691,7 @@ static TmEcode UnixManagerUptimeCommand(json_t *cmd,
                                  json_t *server_msg, void *data)
 {
     SCEnter();
-    int uptime;
+    int64_t uptime;
```

Comment:
why not `time_t`? It is the type for both `time(NULL)` and `ucmd->start_timestamp`

Reply:
right